README.md

```diff
@@ -27,6 +27,18 @@ Logs:
 ```
 
 ### Examples
+##### Express
+```var express = require('express');
+var app = express();
+var KnockKnock = require('./lib/knock-knock');
```

Comment:
Actually, I think you should use `@modulus/standard` style, and use:

`const Express` and `const KnockKnock`